mirage/concept-fixtures/dummy.js

```
@@ -6,7 +6,7 @@ export default {
       blocks: [
         { args: { markdown: 'Hello, world!' }, type: 'markdown' },
         { args: {}, type: 'click_to_continue' },
-        { args: { markdown: 'Hello world 2!' }, type: 'markdown' },
+        { args: { markdown: '### Variable assignment with `if` expressions\n\nYou can also assign the result of an `if` expression.\n\nIn the example below, `weather` is getting assigned by the result of the `if` expression.\n\n```rust\nlet temperature = 15;\n\nlet weather = if temperature <= 10 {\n      "cold"\n} else if temperature <= 20 {\n      "cool"\n} else {\n      "warm"\n};,\n\nprintln!("The weather is {}", weather); // "The weather is cool"\n```' }, type: 'markdown' },
```

ESLint

[error] Replace `·args:·{·markdown:·'###·Variable·assignment·with·`if`·expressions\n\nYou·can·also·assign·the·result·of·an·`if`·expression.\n\nIn·the·example·below,·`weather`·is·getting·assigned·by·the·result·of·the·`if`·expression.\n\n```rust\nlet·temperature·=·15;\n\nlet·weather·=·if·temperature·<=·10·{\n······"cold"\n}·else·if·temperature·<=·20·{\n······"cool"\n}·else·{\n······"warm"\n};,\n\nprintln!("The·weather·is·{}",·weather);·//·"The·weather·is·cool"\n```'·},·type:·'markdown'` with `⏎··········args:·{⏎············markdown:⏎··············'###·Variable·assignment·with·`if`·expressions\n\nYou·can·also·assign·the·result·of·an·`if`·expression.\n\nIn·the·example·below,·`weather`·is·getting·assigned·by·the·result·of·the·`if`·expression.\n\n```rust\nlet·temperature·=·15;\n\nlet·weather·=·if·temperature·<=·10·{\n······"cold"\n}·else·if·temperature·<=·20·{\n······"cool"\n}·else·{\n······"warm"\n};,\n\nprintln!("The·weather·is·{}",·weather);·//·"The·weather·is·cool"\n```',⏎··········},⏎··········type:·'markdown',⏎·······`

(prettier/prettier)